### PR TITLE
fix some syntax errors

### PR DESCRIPTION
**Pull request by Indjov**
fix some syntax errors

### CODE REVIEW

**Comment by Zarthus** [comment]
Your diff is extremely large in in some situations removes translations. I don't think this is intended.

File: po/bg.po [hunk 1]
```diff
@@ -1,270 +1,83 @@
...
-msgstr "%s е замразен."
+msgstr ""
 
-#: modules/botserv/bottalk.c:86 modules/botserv/bottalk.c:152
+#: modules/botserv/bottalk.c:86 modules/botserv/bottalk.c:152 
 #: modules/botserv/main.c:954
 #, c-format
 msgid "%s does not have a bot assigned."
-msgstr "%s няма назначен бот."
+msgstr ""
```
Comment (Zarthus) [inline po/bg.po]: Translation was removed. (I see at least 9 different cases in here without inflicting self harm due to browser speed)

**Comment by ilbelkyr** [comment]
@Indjov any chance you could address @Zarthus's comment? I'm not familiar with the language myself so can't really say much more than he already did.

**Comment by Indjov** [comment]
@ilbelkyr , i will make a new translate.

**Pull request closed without merging**